kitsune/sumo/static/sumo/js/form-wizard-reminder-dialog.js

```diff
@@ -143,9 +131,9 @@ export class ReminderDialog extends HTMLDialogElement {
    *   The description for the event, including the linkURL.
    */
   #generateEventSummaryAndDescription(linkURL, format) {
-    let summary = gettext("Reminder to complete your Firefox backup");
+    let summary = gettext("Download and install Firefox on your new device");
     let description = interpolate(
-      gettext("Your Firefox data has been successfully backed up.\n\nFollow this link to start your download: %s"),
+      gettext(`Your Firefox data is successfully backed up. To get started on your new device, download Firefox and sign in to your account with the link below.\n\n<a href="%s">Download Firefox</a>`),
```

Comment:
It looks like for `.ics` we might be better off omitting the `<a>` tag since the calendar app on mac seems to be smart about linkifying things

<img width="295" alt="Screenshot 2023-10-30 at 1 16 28 PM" src="https://github.com/mozilla/kitsune/assets/15138046/62c9ab8d-9ad3-4909-976e-d69be63874d2">

Reply:
Good call. I'll special-case the iCal case, and just use the URL in that case.